### DISCUSSION TITLE
figma kit 2.1.1 : Check panel, togglenutton publishing

### DISCUSSION THREAD

**Discussion by withroni** [primefaces/community#1688] [Figma UI Kit]
I want to use panel, togglebutton component on my project, but it's invaild assets on pulish library of primeone-2.1.1
please check this. and attached images
![image](https://github.com/primefaces/community/assets/105827321/903dfe31-99ea-4e85-825f-50885635123d)
![image](https://github.com/primefaces/community/assets/105827321/a2f63992-c4c0-4b1d-9e17-3931a5768529)
![image](https://github.com/primefaces/community/assets/105827321/53384b28-ef1e-4fa3-8747-e21719d82be8)

**Comment by onursenture** [comment]
Hello, the issues you mentioned have been fixed in version 2.2.0, but if you prefer to stay on version 2.1.1, please make the following changes before publishing:

• Select the panel component and in the Properties panel, rename the boolean properties defined as Header and Footer to "Show Header" and "Show Footer" respectively.

<img width="1277" alt="Screenshot 2024-04-19 at 18 23 21" src="https://github.com/primefaces/community/assets/244957/ecf97b2f-7ae1-4e44-b65e-41d6f1b1286b">

• Select the toggleButton component and change the first text (which is also a boolean property) in the properties panel to "Show Text."

<img width="1279" alt="Screenshot 2024-04-19 at 18 26 48" src="https://github.com/primefaces/community/assets/244957/9ceeec4e-f4c1-451e-b32c-0e570bdac0ac">

After making these changes, you can publish without any conflicts.
